rpm/libzypp.spec

```diff
@@ -32,6 +35,9 @@ BuildRequires:  libsolv-tools >= 0.7.15
 BuildRequires:  pkgconfig(sigc++-2.0)
 BuildRequires:  glib2-devel
 BuildRequires:  pkgconfig(popt)
+BuildRequires:  pkgconfig(protobuf)
+BuildRequires:  pkgconfig(protobuf-lite)
+BuildRequires:  pkgconfig(readline)
```

Comment:
Is this mandatory here? We have dropped readline by purpose to reduce dependencies from our core image.

Reply:
Grepping readline it looks like used only in zypp-tui and that having a static library. Wasn't able to confirm due to build failure, but perhaps that doesn't end up as dependency on the resulting main package(?)

Reply:
It does not end up as dependency of the main package which gets installed on device, only devel package will depend on it